src/test/components/TooltipCallnode.test.js

```diff
@@ -63,4 +66,84 @@ describe('TooltipCallNode', function() {
 
     expect(container.firstChild).toMatchSnapshot();
   });
+
+  it('displays Page URL for non-iframe pages', () => {
+    const {
+      profile,
+      funcNamesDictPerThread: [funcNamesDict],
+    } = getProfileFromTextSamples(`
+      A
+      Bjs
+      Cjs
+    `);
+    const threadIndex = 0;
+    // Add an item to Pages array.
+    const pageUrl = 'https://developer.mozilla.org/en-US/';
+    profile.pages = [
+      {
+        browsingContextID: 1,
+        innerWindowID: 123123,
+        url: pageUrl,
+        embedderInnerWindowID: 0,
+      },
+    ];
+
+    const { frameTable } = profile.threads[threadIndex];
+
+    for (let i = 1; i < frameTable.length; i++) {
+      frameTable.innerWindowID[i] = profile.pages[0].innerWindowID;
+    }
+
+    const callNodePath = ['A', 'Bjs', 'Cjs'].map(name => funcNamesDict[name]);
+    const { dispatch, renderTooltip } = setup(profile);
+    dispatch(changeSelectedCallNode(threadIndex, callNodePath));
+    const { container, getByText } = renderTooltip();
+
+    expect(getByText(pageUrl)).toBeTruthy();
+    expect(container.firstChild).toMatchSnapshot();
```

Comment:
I checked with both snapshot and `getByText` function in those tests. Probably one of them is enough but wanted to keep both of them. My reasoning was, `getByText` makes sure we have that URL in place(in case we miss to check the snapshot changes in the future) and snapshot makes sure that the URLs have the correct titles(Page URL-iframe URL).